src/style.css

```diff
@@ -1 +1,69 @@
-/* Styles go here */
+:root {
+  --color-hover: #00acdc;
+  --font-size-main: 12px;
+  --color-main: #000;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 100%;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__item:not(:last-child) {
```

Comment:
not fixed 
<img width="728" alt="image" src="https://user-images.githubusercontent.com/102411130/232988619-b4419898-27cf-4bd0-b160-689a8c2235d0.png">